src/Versions/V2_1_1/Client/Credentials.php

```diff
@@ -4,26 +4,81 @@
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Delete\DeleteCredentialsService;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Get\GetCredentialsRequest;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Get\GetCredentialsResponse;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Get\GetCredentialsService;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Post\PostCredentialsRequest;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Post\PostCredentialsResponse;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Post\PostCredentialsService;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Put\PutCredentialsRequest;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Put\PutCredentialsResponse;
+use Chargemap\OCPI\Versions\V2_1_1\Client\Credentials\Put\PutCredentialsService;
 
 class Credentials extends AbstractFeatures
 {
-    private ?RegisterCredentialsService $registerCredentialsService = null;
+    private ?PostCredentialsService $postCredentialsService = null;
+    private ?GetCredentialsService $getCredentialsService = null;
+    private ?PutCredentialsService $putCredentialsService = null;
+    private ?DeleteCredentialsService $deleteCredentialsService = null;
 
     /**
-     * @throws \Chargemap\OCPI\Common\Client\Modules\Credentials\Register\ClientAlreadyRegisteredException
+     * @throws \Chargemap\OCPI\Common\Client\Modules\Credentials\ClientAlreadyRegisteredException
+     * @throws \Chargemap\OCPI\Common\Client\OcpiEndpointNotFoundException
+     * @throws \Chargemap\OCPI\Common\Server\Errors\OcpiInvalidPayloadClientError
      * @throws \Psr\Http\Client\ClientExceptionInterface
+     */
+    public function register(PostCredentialsRequest $request): PostCredentialsResponse
+    {
+        if ($this->postCredentialsService === null) {
+            $this->postCredentialsService = new PostCredentialsService($this->ocpiConfiguration);
+        }
+
+        return $this->postCredentialsService->handle($request);
+    }
+
+    /**
+     * @throws \Chargemap\OCPI\Common\Client\OcpiEndpointNotFoundException
      * @throws \Chargemap\OCPI\Common\Server\Errors\OcpiInvalidPayloadClientError
+     * @throws \Psr\Http\Client\ClientExceptionInterface
+     */
+    public function get(GetCredentialsRequest $request): GetCredentialsResponse
+    {
+        if ($this->getCredentialsService === null) {
+            $this->getCredentialsService = new GetCredentialsService($this->ocpiConfiguration);
+        }
+
+        return $this->getCredentialsService->handle($request);
+    }
+
+    /**
+     * @throws \Chargemap\OCPI\Common\Client\Modules\Credentials\ClientNotRegisteredException
+     * @throws \Chargemap\OCPI\Common\Client\OcpiEndpointNotFoundException
+     * @throws \Chargemap\OCPI\Common\Server\Errors\OcpiInvalidPayloadClientError
+     * @throws \Psr\Http\Client\ClientExceptionInterface
+     */
+    public function update(PutCredentialsRequest $request): PutCredentialsResponse
```

Comment:
This could be `put` to match the other clients

Reply:
Good idea, have fixed it now